### PR TITLE
Added gitmoji.dev keywords

### PR DESCRIPTION
**Pull request by BreakingTV**
These keywords are just quickly thought out in my head. If you have any improvements or other requests for changes, feel free to review!

### CODE REVIEW

**Comment by BreakingTV** [comment]
Closes: #230

**Review by muan — CHANGES_REQUESTED** [review]
Hi, could you uncheck the package-lock.json changes?

**Comment by BreakingTV** [comment]
@muan done!

**Comment by BreakingTV** [comment]
was a bit to fast... removed the entire package-lock...
but now it should be done! @muan

File: dist/emoji-en-US.json [hunk 1]
```diff
@@ -765,7 +765,9 @@
     "face",
     "stuffy",
     "wealthy",
-    "rich"
+    "rich",
+    "data_exploration",
+    "data_inspection"
```
Comment (muan) [inline dist/emoji-en-US.json:770]:
```suggestion
    "exploration",
    "inspection"
```

File: dist/emoji-en-US.json [hunk 2]
```diff
@@ -1158,11 +1160,14 @@
     "face",
     "monster",
     "poop",
-    "smiling"
+    "smiling",
+    "bad_code",
```
Comment (muan) [inline dist/emoji-en-US.json:1164]:
```suggestion
    "bad",
```

File: dist/emoji-en-US.json [hunk 3]
```diff
@@ -1227,7 +1232,8 @@
     "fairy",
     "fantasy",
     "monster",
-    "tale"
+    "tale",
+    "external_api",
```
Comment (muan) [inline dist/emoji-en-US.json:1236]:
```suggestion
    "external",
```

File: dist/emoji-en-US.json [hunk 4]
```diff
@@ -1403,7 +1409,9 @@
     "mute",
     "not",
     "no speaking",
-    "prohibited"
+    "prohibited",
+    "add_gitignore",
+    "update_gitignore"
```
Comment (muan) [inline dist/emoji-en-US.json:1414]:
```suggestion
    "prohibited",
    "ignore"
```

File: dist/emoji-en-US.json [hunk 5]
```diff
@@ -1569,7 +1577,8 @@
     "like",
     "affection",
     "valentines",
-    "nct"
+    "nct",
+    "fix_ci_build"
```
Comment (muan) [inline dist/emoji-en-US.json:1581]:
```suggestion
    "nct"
```

File: dist/emoji-en-US.json [hunk 6]
```diff
@@ -1640,7 +1649,8 @@
     "impact",
     "red",
     "spark",
-    "symbol"
+    "symbol",
+    "breaking_changes"
```
Comment (muan) [inline dist/emoji-en-US.json:1653]:
```suggestion
    "break"
```

File: dist/emoji-en-US.json [hunk 7]
```diff
@@ -1650,7 +1660,11 @@
     "magic",
     "circle",
     "comic",
-    "symbol"
+    "symbol",
+    "add_animations",
+    "update_animations",
+    "add_transitions",
+    "update_transitions",
```
Comment (muan) [inline dist/emoji-en-US.json:1667]:
```suggestion
    "animations",
    "transitions"
```

File: dist/emoji-en-US.json [hunk 8]
```diff
@@ -1704,7 +1718,11 @@
     "chat",
     "comic",
     "comment",
-    "dialog"
+    "dialog",
+    "add_text",
+    "update_text",
+    "add_literals",
+    "update_literals"
```
Comment (muan) [inline dist/emoji-en-US.json:1725]:
```suggestion
    "text",
    "literals"
```

File: dist/emoji-en-US.json [hunk 9]
```diff
@@ -3263,7 +3281,8 @@
     "engineer",
     "laptop",
     "software",
-    "technology"
+    "technology",
+    "improve_developer_experience"
```
Comment (muan) [inline dist/emoji-en-US.json:3285]:
```suggestion
    "developer"
```

File: dist/emoji-en-US.json [hunk 10]
```diff
@@ -5491,7 +5512,9 @@
     "shadows",
     "silhouettes",
     "two",
-    "users"
+    "users",
+    "add_contributors",
+    "update_contributor"
```
Comment (muan) [inline dist/emoji-en-US.json:5517]:
```suggestion
    "contributors"
```

File: dist/emoji-en-US.json [hunk 11]
```diff
@@ -6497,7 +6520,9 @@
     "spring",
     "sprout",
     "sprouting",
-    "young"
+    "young",
+    "add_seed_files",
+    "update_seed_files"
```
Comment (muan) [inline dist/emoji-en-US.json:6525]:
```suggestion
    "seed"
```

File: dist/emoji-en-US.json [hunk 12]
```diff
@@ -7068,7 +7093,9 @@
     "egg",
     "food",
     "chicken",
-    "breakfast"
+    "breakfast",
+    "update_easter_egg",
+    "add_easter_egg"
```
Comment (muan) [inline dist/emoji-en-US.json:7098]:
```suggestion
    "easter_egg"
```

File: dist/emoji-en-US.json [hunk 13]
```diff
@@ -7152,7 +7179,9 @@
     "food",
     "japanese",
     "box",
-    "lunch"
+    "lunch",
+    "add_assets",
+    "update_assets"
```
Comment (muan) [inline dist/emoji-en-US.json:7184]:
```suggestion
    "assets"
```

a bit far fetched but eh.

File: dist/emoji-en-US.json [hunk 14]
```diff
@@ -7606,7 +7635,8 @@
     "cheers",
     "clink",
     "drinks",
-    "mug"
+    "mug",
+    "write_drunkenly"
```
Comment (muan) [inline dist/emoji-en-US.json:7639]:
```suggestion
    "mug"
```

drunk is already a keyword.

File: dist/emoji-en-US.json [hunk 15]
```diff
@@ -7883,15 +7915,17 @@
     "wip",
     "working",
     "progress",
-    "crane"
+    "crane",
+    "architectural_changes"
```
Comment (muan) [inline dist/emoji-en-US.json:7919]:
```suggestion
    "architectural"
```

File: dist/emoji-en-US.json [hunk 16]
```diff
@@ -7883,15 +7915,17 @@
...
   ],
   "🧱": [
     "brick",
     "bricks",
     "clay",
     "construction",
     "mortar",
-    "wall"
+    "wall",
+    "infrastructure_changes"
```
Comment (muan) [inline dist/emoji-en-US.json:7928]:
```suggestion
    "infrastructure"
```

File: dist/emoji-en-US.json [hunk 17]
```diff
@@ -8445,7 +8479,8 @@
     "health",
     "911",
     "hospital",
-    "vehicle"
+    "vehicle",
+    "hotfix"
```
Comment (muan) [inline dist/emoji-en-US.json:8483]:
```suggestion
    "vehicle"
```

Sorry I don't think this is general enough.

File: dist/emoji-en-US.json [hunk 18]
```diff
@@ -8526,7 +8561,9 @@
     "delivery_truck",
     "cars",
     "transportation",
-    "vehicle"
+    "vehicle",
+    "move_resources",
+    "rename_resources"
```
Comment (muan) [inline dist/emoji-en-US.json:8566]:
```suggestion
    "vehicle",
    "resources"
```

File: dist/emoji-en-US.json [hunk 19]
```diff
@@ -8654,7 +8691,9 @@
     "revolving",
     "rotating",
     "siren",
-    "vehicle"
+    "vehicle",
+    "fix_compiler_warning",
+    "fix_linter_warning"
```
Comment (muan) [inline dist/emoji-en-US.json:8696]:
```suggestion
    "vehicle",
    "warning"
```

File: dist/emoji-en-US.json [hunk 20]
```diff
@@ -9717,7 +9758,8 @@
     "electric",
     "electricity",
     "sign",
-    "thunderbolt"
+    "thunderbolt",
+    "improve_performance"
```
Comment (muan) [inline dist/emoji-en-US.json:9762]:
```suggestion
    "thunderbolt",
    "speed"
```

File: dist/emoji-en-US.json [hunk 21]
```diff
@@ -9864,7 +9907,8 @@
     "entertainment",
     "glitter",
     "sparkle",
-    "star"
+    "star",
+    "new_feature"
```
Comment (muan) [inline dist/emoji-en-US.json:9911]:
```suggestion
    "star"
```

File: dist/emoji-en-US.json [hunk 22]
```diff
@@ -9888,7 +9932,8 @@
     "activity",
     "entertainment",
     "hat",
-    "hooray"
+    "hooray",
+    "begin_project"
```
Comment (muan) [inline dist/emoji-en-US.json:9936]:
```suggestion
    "hooray"
```

File: dist/emoji-en-US.json [hunk 23]
```diff
@@ -10243,7 +10288,8 @@
   ],
   "🥅": [
     "goal_net",
-    "sports"
+    "sports",
+    "catch_errors"
```
Comment (muan) [inline dist/emoji-en-US.json:10292]:
```suggestion
    "sports",
    "catch"
```

File: dist/emoji-en-US.json [hunk 24]
```diff
@@ -10553,15 +10599,21 @@
     "art",
     "entertainment",
     "museum",
-    "painting"
+    "painting",
+    "improve_structure",
+    "improve_format"
```
Comment (muan) [inline dist/emoji-en-US.json:10604]:
```suggestion
    "improve"
```

File: dist/emoji-en-US.json [hunk 25]
```diff
@@ -10553,15 +10599,21 @@
...
     "needle",
     "sewing",
     "spool",
     "string",
-    "crafts"
+    "crafts",
+    "add_multithreading",
+    "update_multithreading",
+    "add_concurrency",
+    "update_concurrency"
```
Comment (muan) [inline dist/emoji-en-US.json:10616]:
```suggestion
    "crafts"
```

File: dist/emoji-en-US.json [hunk 26]
```diff
@@ -13096,7 +13201,8 @@
     "cardinal",
     "north",
     "pointing",
-    "upwards"
+    "upwards",
+    "upgrade_dependencies"
```
Comment (muan) [inline dist/emoji-en-US.json:13205]:
```suggestion
    "upgrade"
```

File: dist/emoji-en-US.json [hunk 27]
```diff
@@ -13144,7 +13250,8 @@
     "downwards",
     "down arrow",
     "pointing",
-    "south"
+    "south",
+    "downgrade_dependencies"
```
Comment (muan) [inline dist/emoji-en-US.json:13254]:
```suggestion
    "downgrade"
```

File: dist/emoji-en-US.json [hunk 28]
```diff
@@ -14018,7 +14133,8 @@
     "subtract",
     "less",
     "heavy",
-    "symbol"
+    "symbol",
+    "remove_dependency"
```
Comment (muan) [inline dist/emoji-en-US.json:14137]:
```suggestion
    "remove"
```

**Review by muan — DISMISSED** [review]
Hey, I ended up having to generalize most of this. I am not sure what your use case for this library is, but this is not meant to be `:emoji_short_code:`. 

If you want to keep the specificity I'd suggest that you keep your own list and add on top of `emojilib`.

**Comment by BreakingTV** [comment]
Completly fine with these suggestions ^^
My bad for not generalizing them, but I think this way its even better!

**Comment by muan** [comment]
Thanks. I'll probably take another pass before publishing. I'll let you know when it's released.

**Pull request merged**